testdata/telegraf-agent.toml

```diff
@@ -23,7 +23,8 @@
 # with 'required'. Be sure to edit those to make this configuration work.
 
 # Configuration for influxdb server to send metrics to
-[influxdb]
+[outputs]
+[outputs.influxdb]
```

Comment:
maybe I didn't do what I thought we had talked about, my understanding was the config would look like:

```toml
[tags]
dc = "us-east-1"
```